frontend/common/dispatcher/action-constants.js

```diff
@@ -66,8 +65,7 @@ const Actions = Object.assign({}, require('./base/_action-constants'), {
     'RESEND_INVITE': 'RESEND_INVITE',
     'SEARCH_AUDIT_LOG': 'SEARCH_AUDIT_LOG',
     'SEARCH_IDENTITIES': 'SEARCH_IDENTITIES',
-    'SELECT_ENVIRONMENT': 'SELECT_ENVIRONMENT',
-    'SAVE_IDENTITY': 'SAVE_IDENTITY',
+    'SAVE_IDENTITY': 'SAaVE_IDENTITY',
```

Comment:
This looks like a typo?

Reply:
I've fixed the typo [here](https://github.com/Flagsmith/flagsmith/pull/1800/commits/0b6505a3e17068fffcebc09d8050d6a48bbbd30c) but I'm not entirely sure why it (seemingly) didn't break anything in the first place so I'm going to leave this discussion open for now.

Reply:
Yeah, the value is meaningless since it acts like an enum. Will definitely fix as part of PR